### PR TITLE
preserve the stable webhook logger names we had before

### PR DESCRIPTION
**Pull request by dprotaso**
[I dropped the queue name being added to the logger](https://github.com/knative/pkg/pull/1991/files#diff-3c5bb5f48211873c58fcba055dcae2ac7b1958969219e06e1508d76d485dace7L233) in https://github.com/knative/pkg/pull/1991 but forgot to keep the webhook logger names stable

/assign @vagababov

### CODE REVIEW

**Comment by dprotaso** [comment]
blah `TestMetricsExport`

/retest

File: webhook/certificates/controller.go [hunk 1]
```diff
@@ -66,7 +66,8 @@ func NewController(
 		secretlister: secretInformer.Lister(),
 	}
 
-	c := controller.NewImpl(wh, logging.FromContext(ctx), "WebhookCertificates")
+	queueName := "WebhookCertificates"
```
Comment (vagababov) [inline webhook/certificates/controller.go:69]: const

File: webhook/configmaps/controller.go [hunk 2]
```diff
@@ -73,7 +73,8 @@ func NewAdmissionController(
 		wh.registerConfig(configName, constructor)
 	}
 
-	c := controller.NewImpl(wh, logging.FromContext(ctx), "ConfigMapWebhook")
+	queueName := "ConfigMapWebhook"
```
Comment (vagababov) [inline webhook/configmaps/controller.go:76]: ditto

File: webhook/resourcesemantics/conversion/controller.go [hunk 3]
```diff
@@ -117,7 +117,9 @@ func NewConversionController(
 		crdLister:    crdInformer.Lister(),
 	}
 
-	c := controller.NewImpl(r, logging.FromContext(ctx), "ConversionWebhook")
+	logger := logging.FromContext(ctx)
+	queueName := "ConversionWebhook"
```
Comment (vagababov) [inline webhook/resourcesemantics/conversion/controller.go:121]: and so on:)

File: injection/README.md [hunk 4]
```diff
@@ -138,6 +138,7 @@ func NewController(ctx context.Context, cmw configmap.Watcher) *controller.Impl
 		Client:        kubeclient.Get(ctx),
 		ServiceLister: svcInformer.Lister(),
 	}
+  logger = logger.Named("NameOfController")
```
Comment (markusthoemmes) [inline injection/README.md:141]: nit: indenting?

File: webhook/psbinding/README.md [hunk 5]
```diff
@@ -142,6 +142,7 @@ func NewController(
 		Recorder: record.NewBroadcaster().NewRecorder(
 			scheme.Scheme, corev1.EventSource{Component: controllerAgentName}),
 	}
+  logger = logger.Named("GithubBindings")
```
Comment (markusthoemmes) [inline webhook/psbinding/README.md:145]: same as above

File: injection/README.md (hunk 4 above)
Comment (dprotaso) [reply injection/README.md:141]: oh weird - IDE must have broke

Comment (dprotaso) [reply injection/README.md:141]: ah it's a markdown

**Review by vagababov — COMMENTED** [review]
/lgtm

**Pull request merged**